pages/agent/v3/help/_pipeline_upload.md

```diff
@@ -35,6 +35,9 @@ provided, the command looks for the file in the following locations:
 You can also pipe build pipelines to the command allowing you to create
 scripts that generate dynamic pipelines.
 
+>🚧
+> Configuration file uploaded using pipeline upload command has a limit of 500 steps per file. If there are more than 500 steps per file then it should be split into multiple configuration files with less than 500 steps each and uploaded as separate steps.
```

Comment:
It looks like this file is autogenerated, so any edits here will be overridden:

> This file is auto-generated by scripts/update-agent-help.sh, please update the
agent CLI help in https://github.com/buildkite/agent and run the generation
script.

Reply:
@mbelton-buildkite I created PR https://github.com/buildkite/agent/pull/2046. Once that is approved I can update this PR running the script.